stdlib/public/Cxx/CxxDictionary.swift

```diff
@@ -14,7 +14,7 @@
 ///
 /// C++ standard library types such as `std::map` and `std::unordered_map`
 /// conform to this protocol.
-public protocol CxxDictionary<Key, Value> {
+public protocol CxxDictionary where Key: Hashable {
```

Comment:
Why did you remove primary associated types `Key, Value` from the signature? Those are important for the ergonomics of this protocol, they should stay.